app/views/v1/invitations/create.json.jbuilder

```diff
@@ -0,0 +1,3 @@
+json.success true
+json.event @event.id.to_s
+json.invitationNumber @invitations.size
```

Comment:
just to be sure it has created invitations for every email in the
email_list.
although.. now I'm thinking it should remove duplicated email
On Jun 9, 2015 8:59 PM, "GpoNsU" <notifications@github.com> wrote:

> In app/views/v1/invitations/create.json.jbuilder
> <https://github.com/TheTreeHouse/turniquette-api/pull/45#discussion_r32050137>
> :
>
> > @@ -0,0 +1,3 @@
> > +json.success true
> > +json.event @event.id.to_s
> > +json.invitationNumber @invitations.size
>
> Why I want to know the number of invitations?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/TheTreeHouse/turniquette-api/pull/45/files#r32050137>.
>